src/app/public/index.ts

```diff
@@ -1 +1,7 @@
-export * from './src/library.module';
+
+// Split view modules.
+export * from './modules/split-view/split-view.module';
+
+// Split view types.
+export * from './modules/split-view/types/split-view-message';
+export * from './modules/split-view/types/split-view-message-type';
```

Comment:
I don't have a lot of strong feelings on this one. However, it was a deliberate move to break convention. Steve and I talked about being more strict with our exports to avoid issues like cyclical dependencies. In a prior review ☝️ , we decided to remove any component/service/class/etc.. that we did not intend for consumers to use.

Reply:
Ok. We can always add them in later.